TodoList/Extensions/UIViewExtensions.swift

```diff
@@ -13,7 +16,11 @@ extension UIView {
         return nibView
     }
     
-    /// Adds a subview and constrains it to the edges
+    /// Adds a subview and constrains it to the edges.
+    ///
+    /// This function also sets sets false to
+    ///
+    ///     translatesAutoresizingMaskIntoConstraints
```

Comment:
Too many lines for so little information. But in any case, you have to set that property to false when creating programmatic constraints, not sure I'd even mention that.